doc/TSCC.rst

```diff
@@ -1,53 +1,51 @@
-  You can also ssh into  those nodes directly (e.g. if you have :code:`tmux` sessions saved on one of them)
+* This will put you on a node such as `login1.tscc.sdsc.edu` or `login11.tscc.sdsc.edu` or `login2.tscc.sdsc.edu`.
+  You can also ssh into those nodes directly (e.g. if you have :code:`tmux` sessions saved on one of them)
 
 The login nodes are often quite slow because there are too many users on them, and you're not supposed to run code that's
 at all computationally burdensome there. So if you want to use tscc as a workstation, you should immediately try to grab an
-interactive session. :code:`qsub -I -l walltime=<n>:00:00 -l nodes=1:ppn=1 -q home` where :code:`<n>` is the max
-number of hours you plan to work. That will wait till it can find a slot on our home node and then log you into
-that node. (Note: :code:`qsub`'s user interface sucks, if you mistype that command it will hang forever without
-telling you that's what happened. You should see :code:`qsub: waiting for job 22409571.tscc-mgr7.local to start`
-if you typed the command correctly)
+interactive session. :code:`srun --partition=condo --account=ddp268 --pty --nodes=1 --ntasks-per-node=4 -t <n>:00:00 --wait=0 --qos=condo --export=ALL /bin/bash` where :code:`<n>` is the max
+number of hours you plan to work. That will wait till it can find a slot on the condo node and then log you into
+that node.
 
-In the unlikely event that our home node is full of work, this will hang till space opens up (up to 8 hours).
+In the unlikely event that our condo node is full of work, this will hang till space opens up.
 
 Any time your internet connection gets disrupted (depending on your settings, when your computer falls asleep) the 
-interactive session will be killed along with any jobs you were running. AFAIK there is no way to preserve processes
-between interactive sessions (i.e. saving sessions with :code:`tmux` won't work). You can ssh directly back in to 
-the node which your interactive session was on, but the scheduler will kill processes you start, so that's not a soltuion.
+interactive session will be killed along with any jobs you were running. To preserve processes
+between interactive sessions, you should use :code:`tmux` or :ref:`screen <snorlax-screen>` on the login node *before* you start the interactive session.
 
 Filesystem locations
 --------------------
-We have 100TB of space in :code:`/projects/ps-gymreklab`, which is where all of our files are stored. Your personal
-storage directory is :code:`/projects/ps-gymreklab/<user>`. Your home directory for config and the like is
-:code:`/home/<user>`, don't store any large files there.
+We have 100TB of space in :code:`/tscc/projects/ps-gymreklab`, which is where all of our files are stored. Your personal
+storage directory is :code:`/tscc/projects/ps-gymreklab/<user>`. Your home directory for config and the like is
+:code:`/tscc/nfs/home/<user>`, don't store any large files there, since you'll only get 100 GB there.
 
```

Comment:
```suggestion

If you need some extra space just for a few months, consider using your Lustre *scratch* directory (`/tscc/lustre/ddn/scratch/$USER`). Files here are deleted automatically after 90 days but there is more than 2 PB available, shared over all of the users of TSCC. Otherwise, if you simply need some extra space just until your job finishes running, you can refer to `/scratch/$USER/job_$SLURM_JOBID` within your jobscript. This storage will be deleted once your job dies, but it's better than Lustre scratch for I/O intensive jobs.